src/docs/config/overview.md

```diff
@@ -166,6 +167,30 @@ The `preamble` configuration is a `string` that represents a preamble in the mai
 preamble: 'Built with Stencil'
 ```
 
+## sourceMap
+
+*default: `false`*
+
+When set to `true`, sourcemaps will be generated for a project.
+
+```tsx
+sourceMap: true
+```
+
+Sourcemaps create a translation between Stencil components that are written in TypeScript & JSX and JavaScript that is
```

Comment:
What do you think of...

```suggestion
Sourcemaps are an opt-in feature that create a translation between Stencil components that are written in TypeScript/JSX and the resulting JavaScript that is
```

Reply:
This sentence is meant to describe sourcemaps generally, rather than the Stencil specific aspects. The idea here is you shouldn't have to go google 'sourcemap' to figure out what one is (at a high level at least)

Reply:
I think I was more opposed to the changes in the original revision, but I think after the edits to the edits your proposed this works 👍